Resources/config/yui.yml

```diff
@@ -960,3 +961,30 @@ system:
                 ez-app-extension-admin:
                     requires: ['ez-serversideview', 'ez-serversideviewservice', 'ez-app-extension']
                     path: %ez_platformui.public_dir%/js/apps/extensions/ez-app-extension-admin.js
+                ez-testoneview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'testoneview-ez-template'
+                        - 'ez-testoneviewservice'
+                        - 'ez-testonearticlesview'
+                        - 'ez-testoneintroview'
+                        - 'ez-xmltext-view'
+                    path: '%ez_platformui.public_dir%/js/views/ez-testoneview.js'
+                ez-testoneviewservice:
+                    requires: ['ez-viewservice']
+                    path: '%ez_platformui.public_dir%/js/views/services/ez-testoneviewservice.js'
+                testoneview-ez-template:
+                    type: 'template'
+                    path: '%ez_platformui.public_dir%/templates/testoneview.hbt'
+                ez-testonearticlesview:
+                    requires: ['ez-templatebasedview', 'testonearticlesview-ez-template', 'ez-testoneviewservice']
+                    path: '%ez_platformui.public_dir%/js/views/ez-testonearticlesview.js'
+                testonearticlesview-ez-template:
+                    type: 'template'
+                    path: '%ez_platformui.public_dir%/templates/testonearticlesview.hbt'
+                ez-testoneintroview:
+                    requires: ['ez-templatebasedview', 'testoneintroview-ez-template', 'ez-testoneviewservice']
```

Comment:
The same as in there: https://github.com/gbalcewicz/PlatformUIBundle/pull/1/files#r41961103